modules/ocf_kvm/manifests/init.pp

```diff
@@ -42,5 +42,14 @@
       ensure => link,
       links  => manage,
       target => '/opt/share/utils/staff/sys/makevm';
+
+    '/opt/share/kvm':
+      ensure => directory;
+
+    '/opt/share/kvm/makevm-ssh-key':
+      content   => file('/opt/puppet/shares/private/makevm-ssh-key'),
```

Comment:
what if we just generated this? i don't think we really care about having a persistent key between kvm hosts or if we rebuild the hosts?

something like

```puppet
exec { 'ssh-keygen -t ed25519 -f /opt/share/kvm/makevm-ssh-key -N ""':
  creates => '/opt/share/kvm/makevm-ssh-key',
}
```

or whatever